src/world/area_kpa/kpa_114/A9E980.c

```diff
@@ -1,3 +1,7 @@
 #include "kpa_114.h"
 
-INCLUDE_ASM(s32, "world/area_kpa/kpa_114/A9E980", func_80240000_A9E980);
+ApiStatus func_80240000_A9E980(Evt* script, s32 isInitialCall) {
+    sprintf((s8* ) &wMapShapeName, "kpa_112_shape");
+    sprintf((s8* ) &wMapHitName, "kpa_112_hit\0kpa_113\0kpa_115");
```

Comment:
This string is just "kpa_112_hit", and the other strings can be separated out and put in const char* vars after this function (see this discussion: https://github.com/pmret/papermario/pull/754#discussion_r958225292)